interactions.go

```diff
@@ -35,12 +35,14 @@ type ApplicationCommand struct {
 	Version           string                 `json:"version,omitempty"`
 	Type              ApplicationCommandType `json:"type,omitempty"`
 	Name              string                 `json:"name"`
+	NameLocalizations *map[Locale]string     `json:"name_localizations,omitempty"`
 	DefaultPermission *bool                  `json:"default_permission,omitempty"`
 
 	// NOTE: Chat commands only. Otherwise it mustn't be set.
 
-	Description string                      `json:"description,omitempty"`
-	Options     []*ApplicationCommandOption `json:"options"`
+	Description              string                      `json:"description,omitempty"`
+	DescriptionLocalizations *map[Locale]string          `json:"description_localizations,omitempty"`
```

Comment:
I'm going to do a separate PR on that. For now it will stay as pointers, `&map[Locale]string` doesn't look too bad.

Reply:
Well I now just noticed it while using localizations...
For me I write a function that returns a pointer to a map, because you can't pass a pointer to a function. And thats not a problem at all.
But I also noticed that `ApplicationCommandOption` and `ApplicationCommandOptionChoice` does not have pointers. While I don't really have a clue why `ApplicationCommand` do, I wondered why the others do not.